frappe/patches/v13_0/replace_old_data_import.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2020, Frappe Technologies Pvt. Ltd. and Contributors
+# MIT License. See license.txt
+
+from __future__ import unicode_literals
+import frappe
+
+
+def execute():
+	frappe.db.sql(
```

Comment:
```suggestion
	frappe.reload_doc("core", "doctype", "data_import_beta")
	frappe.db.sql(
```

to avoid
<img width="670" alt="Screenshot 2020-06-17 at 9 51 54 AM" src="https://user-images.githubusercontent.com/13928957/84859651-35561c80-b08b-11ea-8a34-0a5c79a39684.png">